packages/theme-ui-expansion-pack/src/Accordion/AccordionIcon.tsx

```diff
@@ -12,9 +13,9 @@ export const AccordionIcon = forwardRef<undefined, BoxProps>((props, ref) => {
       data-testid="accordion-icon"
       aria-hidden
       {...props}
-      __css={{
+      css={{
```

Comment:
__css baza `ts` error, može se staviti `// @ts-ignore` liniju iznad, ali zašto bi to bilo bolje rješenje?

Reply:
al nije isto...
`__css` se koristi u Box komponenti i dolazi prije `sx` stilova https://github.com/system-ui/theme-ui/blob/master/packages/components/src/Box.js#L13

Druga stvar, ako zelis koristiti tokene is teme onda moras provuc kroz theme-ui `css` helper funkciju https://theme-ui.com/api#css

Reply:
> __css se koristi u Box komponenti i dolazi prije sx stilova

Onda solucija sa `// @ts-ignore` ?

> Druga stvar, ako zelis koristiti tokene is teme onda moras provuc kroz theme-ui css helper funkciju

Nice, nisam znao za ovo 😄

Reply:
da `__css` za sad